Sources/SkipFoundation/UserDefaults.swift

```diff
@@ -350,6 +383,11 @@ public class UserDefaults: KotlinConverting<android.content.SharedPreferences> {
         fatalError()
     }
 
+    enum UnrepresentableType: Int {
+        case unspecified, double, date, data, url
```

Comment:
Can you add explicit values here like: `case unspecified = 0, double = 1, date = 2, data = 3, url = 4`, just to hint to any future additions (like `array`) that they need to maintain compatibility with existing values?